includes/datavalues/SMW_DataValue.php

```diff
@@ -968,8 +968,10 @@ abstract protected function loadDataItem( SMWDataItem $dataItem );
 	 * Overwritten by callers to supply an array of parameters that can be used for
 	 * creating servicelinks. The number and content of values in the parameter array
 	 * may vary, depending on the concrete datatype.
+	 *
+	 * @return bool|array
 	 */
-	protected function getServiceLinkParams(): bool {
+	protected function getServiceLinkParams() {
```

Comment:
This is a manual fix where Rector didn't understand that the subclasses return arrays.